xds/googledirectpath/googlec2p.go

```diff
@@ -106,25 +102,21 @@ func (c2pResolverBuilder) Build(t resolver.Target, cc resolver.ClientConn, opts
 	if balancerName == "" {
 		balancerName = tdURL
 	}
-	serverConfig, err := bootstrap.ServerConfigFromJSON([]byte(fmt.Sprintf(`
+
+	node := newNode(<-zoneCh, <-ipv6CapableCh)
+	xdsServer := newXdsServer(balancerName)
+	authorities := newAuthorities(xdsServer)
```

Comment:
optional: how about `new(Whatever)Config`? `newXdsServer` sounds to me like we are creating something new here. But essentially its only the config

Reply:
Done. Good insight!